react/features/base/tracks/actions.js

```diff
@@ -351,17 +351,16 @@ function _getLocalTracksToChange(currentTracks, newTracks) {
 /**
  * Mutes or unmutes a specific <tt>JitsiLocalTrack</tt>. If the muted state of
  * the specified <tt>track</tt> is already in accord with the specified
- * <tt>muted</tt> value, then does nothing. In case the actual muting/unmuting
- * fails, a rollback action will be dispatched to undo the muting/unmuting.
+ * <tt>muted</tt> value, then does nothing.
  *
  * @param {JitsiLocalTrack} track - The <tt>JitsiLocalTrack</tt> to mute or
  * unmute.
  * @param {boolean} muted - If the specified <tt>track</tt> is to be muted, then
  * <tt>true</tt>; otherwise, <tt>false</tt>.
- * @returns {Function}
+ * @returns {Promise}
```

Comment:
It returns a function which returns a promise, really. But that I think of it, you could probably not do that and just do the thing and return the promise indeed.